.github/workflows/web_docs.yaml

```diff
@@ -0,0 +1,46 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy API Documentation
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches:
+      - main
+      - alpha
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
```

Comment:
Since we don't use OIDC auth in this workflow, we can probably remove the `id-token` permission I think, apologies for not picking this up earlier

Reply:
Good point. Removed!